docs/guides/burst_insar_product_guide.md

```diff
@@ -221,35 +229,102 @@ The text file with extension .README.md.txt explains the files included in the f
+├─ S1_136232_IW2_20200604_20200616_VV_INT80_663F
+├─ S1_136231_IW2_20200604_20200616_VV_INT80_529D
+```
+
+In order to be merging eligible, all burst products must:
+
+- Have the same reference and secondary dates
+- Have the same polarization
+- Have the same multilooking
+- Be from the same relative orbit
+- Be contiguous
+
+The workflow should throw an error if any of these conditions are not met.
+
+#### Merge Processing
+During normal ISCE2 InSAR processing, initial interferograms are formed on a burst-by-burst basis. These range-doppler burst interferograms are combined during an ISCE2 step called `merge_bursts`, then the remaining steps (filtering, unwrapping and geocoding) are conducted on the merged results.
+
+By including select range-doppler data (unwrapped interferogram, geolocation information, and line-of-sight information) as well as select metadata in our standard Burst InSAR products, we are able to restart ISCE2 processing from the `merge_burst` step, then proceed with the following steps as if it were a standard ISCE2 InSAR processing run.
+
+The steps of the workflow are as follows:
+
+1.	Recreate a pre-`merge_bursts` ISCE2 InSAR processing state using the input Burst InSAR products.
+2.	Run a modified version of ISCE2’s `merge_burst` step.
+3.	Apply the Goldstein-Werner power spectral filter with a dampening factor of 0.5.
+4.	Unwrap the wrapped phase interferogram using [SNAPHU](http://web.stanford.edu/group/radar/softwareandlinks/sw/snaphu/){target=_blank}'s minimum cost flow (MCF) unwrapping algorithm to produce the unwrapped phase interferogram.
+5.	Geocode the output products.
+
+As mentioned above, this workflow uses underlying ISCE2 functionality to perform these steps so the results of this workflow should be identical to the results obtained by performing a standard multi-burst ISCE2 InSAR run (assuming that the Enhanced Spectral Diversity technique is not used for co-registration).
+
+#### Merge Processing Options
+
+The processing options available for the merging are the same as those available for standard Burst InSAR products. Check out the [Processing Options](#processing-options "Jump to the Processing Options section of this document") section for more details.
+
+To learn about the command line argument syntax for this workflow, look at the help documentation using:
+```bash
+python -m hyp3_isce2 ++process merge_tops_bursts --help
+```
+
+#### Product Packaging
+The product packaging of merged burst InSAR products follows the same conventions outlined in [Product Packaging](#product-packaging "Jump to the Product Packaging section of this document") section above with two exceptions. First, the four range-doppler images are not included since the products have already been merged. Second, the product is modified slightly; the Burst ID is swapped for the zero-padded relative orbit number, and the swath number are removed. The resulting format is:
+```
+S1_rrr__yyymmdd_yyymmdd_pp_INTn_uuuu
+```
```

Comment:
Do we need to list the file names in the merge product as a table and also give the example filenames just like in Table 2 ?

Reply:
@cirrusasf I can see the logic in this, but I don't think we should. Adding a second table would add a ton of duplication to the product guide and I think we're better off just referencing the product packaging section like we've done here.